experimental/globaldata/src/data/core_helloworld_v1/mod.rs

```diff
@@ -0,0 +1,34 @@
+include!("bn.data.rs");
+include!("en.data.rs");
+include!("ja.data.rs");
+include!("ru.data.rs");
+
+#[macro_export]
+macro_rules! impl_core_helloworld_v1 {
```

Comment:
What's the plan for `AnyProvider`? Having one macro per key doesn't work, the keys should be arguments (plural) to the macro.

Reply:
I think it should "just work"

```rust
struct LocalBakedProvider;
impl_core_helloworld_v1!(LocalBakedProvider);
impl_decimal_symbols_v1!(LocalBakedProvider);
impl_any_provider!(LocalBakedProvider, [
  icu_provider::hello_world::HelloWorldV1Marker,
  icu_decimal::provider::SymbolsV1Marker,
]);
```